pallets/compliance-manager/src/benchmarking.rs

```diff
@@ -0,0 +1,363 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+use pallet_asset::SecurityToken;
+use pallet_balances as balances;
+use polymesh_common_utilities::asset::AssetType;
+use polymesh_primitives::{IdentityId, InvestorUid, TrustedFor, TrustedIssuer};
+
+use frame_benchmarking::{account, benchmarks};
+use frame_support::traits::Currency;
+use frame_system::RawOrigin;
+
+const SEED: u32 = 1;
+
+/// Helper class to create accounts and its DID to simplify benchmarks and UT.
+pub struct User<T: Trait> {
+    pub account: T::AccountId,
+    pub origin: RawOrigin<T::AccountId>,
+    pub did: Option<IdentityId>,
+}
+
+impl<T: Trait> User<T> {
+    /// It creates an account based on `name` and `u` with 1_000_000 as free balance.
```

Comment:
For all documentation added, prefer describing functions and methods as verbs in present tense.
E.g., "Create an account..." in this case.